R/nested.R

```diff
@@ -24,12 +24,13 @@
 #'     - Function calls
 #'     - Function definitions
 #' - Remove `includeText = TRUE`
+#' @importFrom purrr rerun
 compute_parse_data_nested <- function(text) {
   parsed <- parse(text = text, keep.source = TRUE)
   parse_data <- tbl_df(utils::getParseData(parsed, includeText = TRUE))
   pd_nested <-
     parse_data %>%
-    mutate_(child = ~rep(list(NULL), length(text))) %>%
+    mutate_(child = ~rerun(length(text), tibble())) %>%
```

Comment:
The `unlist(...)` call looks expensive. Why not just check `nrow()` for the tibble returned from `bind_rows()`?

A benchmark seems low priority to me. I think we'll be much faster when we start using `getParseData(includeText = NA)` again. But we could compare the code for both solutions.